RequestService/RequestService.Repo/Repository.cs

```diff
@@ -2121,16 +2121,38 @@ public async Task<Chart> GetActivitiesByMonth(int groupId)
                     Count = s.Count()
                 }).ToList();
 
-            var x = chartItems.GroupBy(g => new { g.Label, date = $"{g.Date:yyyy}-{g.Date:MM}" })
+            //Get distinct activities from list
+            var supportActivities = chartItems.Select(x => x.Label).Distinct().ToList();
+
+            //Populate chartitems with support activities for each month
+            for (int i = 0; i < 12; i++)
```

Comment:
Hard-coded 12 doesn't seem ideal.  Could we compare with DateTime.UtcNow instead?